### ISSUE TITLE
Layout/EmptyLinesAroundArguments false positive

### ISSUE THREAD

**Issue by rhys-vdw** [rubocop-hq/rubocop#6001]
No idea what's going on here

--------

## Expected behavior

No "empty lines around arguments error here:

```rb
class AnswerTags

  # ... REDACTED ...

  private_class_method def self.create_answer_tags(question_id:, question_tag_id:, unfiltered_answer_ids:)
    answer_ids = if unfiltered_answer_ids.present?
      # Filter answer IDs to exclude anything from other questions/accounts.
      ResponseAnswer
        .where(
          usability_test_section_question_id: question_id,
          id: unfiltered_answer_ids,
        )
        .pluck(:id)
    else
      []
    end
               # ... This is line 102 ...
    if answer_ids.any?
      # Manually inserting tags to take advantage of PostgreSQL's upsert
      # functionality. Note that conflicts result in a no-op update to ensure
      # that all rows are returned.
      now = Time.zone.now
      result = AnswerTag.find_by_sql [
        <<~SQL,
          insert into answer_tags (question_tag_id, response_answer_id, created_at, updated_at)
            values #{answer_ids.map { '(?, ?, ?, ?)' }.join(', ')}
          on conflict (question_tag_id, response_Answer_id)
            do update set question_tag_id = excluded.question_tag_id
          returning *
        SQL
        *answer_ids.flat_map { |answer_id| [question_tag_id, answer_id, now, now] }
      ]
      ModelSerialization.serialize(result, AnswerTag::PUBLIC_READ_ATTRIBUTES)
    else
      []
    end
  end
end
```

## Actual behavior

```
app/services/answer_tags.rb:102 W: Layout/EmptyLinesAroundArguments: Empty line detected around arguments.
```

## Steps to reproduce the problem

The code

## RuboCop version

```
-> bundle exec rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by koic** [comment]
Thanks for reporting. 
I could not reproduce. Would you create a minimum and entire code to reproduce your issue.

**Comment by bbatsov** [comment]
@rhys-vdw Can you try this with some recent RuboCop - e.g. 0.57.2?

**Comment by rhys-vdw** [comment]
Will do so in a week.
